resources/lang/de/command/messages.php

```diff
@@ -5,80 +5,80 @@
-            'connection_error' => 'Unable to connect to the MySQL server using the provided credentials. The error returned was ":error".',
-            'creds_not_saved' => 'Your connection credentials have NOT been saved. You will need to provide valid connection information before proceeding.',
-            'try_again' => 'Go back and try again?',
+            'host_warning' => 'Es wird dringend empfohlen, "localhost" nicht als Datenbank-Host zu verwenden, da wir häufige Socket-Verbindungsprobleme gesehen haben. Wenn du eine lokale Verbindung verwenden willst, solltest du "127.0.0.0.1" verwenden.',
+            'host' => 'Datenbank Host',
+            'port' => 'Datenbank Port',
+            'database' => 'Datenbank Name',
+            'username_warning' => 'Die Verwendung des "root"-Accounts für MySQL-Verbindungen ist nicht nur sehr verpönt, sondern auch von dieser Anwendung nicht erlaubt. Sie müssen einen MySQL-Benutzer für diese Software erstellt haben.',
+            'username' => 'Datenbank Benutzername',
+            'password_defined' => 'Es scheint so, als ob du bereits ein MySQL-Verbindungspasswort definiert hast, möchtest du es ändern?',
+            'password' => 'Datenbank Passwort',
+            'connection_error' => 'Die Verbindung zum MySQL-Server kann mit den angegebenen Anmeldeinformationen nicht hergestellt werden. Der zurückgegebene Fehler war ":error".',
+            'creds_not_saved' => 'Deine Verbindungsdaten wurden NICHT gespeichert. Du musst gültige Verbindungsinformationen angeben, bevor du fortfahren kannst.',
+            'try_again' => 'Zurückgehen und es nochmal versuchen?',
         ],
         'app' => [
-            'app_url_help' => 'The application URL MUST begin with https:// or http:// depending on if you are using SSL or not. If you do not include the scheme your emails and other content will link to the wrong location.',
-            'app_url' => 'Application URL',
-            'timezone_help' => 'The timezone should match one of PHP\'s supported timezones. If you are unsure, please reference http://php.net/manual/en/timezones.php.',
-            'timezone' => 'Application Timezone',
-            'cache_driver' => 'Cache Driver',
-            'session_driver' => 'Session Driver',
-            'using_redis' => 'You\'ve selected the Redis driver for one or more options, please provide valid connection information below. In most cases you can use the defaults provided unless you have modified your setup.',
+            'app_url_help' => 'Die Anwendungs URL MUSS mit https:// oder http:// beginnen, je nach dem ob du SSL verwendest oder nicht. Solltest du das Schema nicht beinhalten, werden deine E-Mails und anderer Inhalt an die falsche Stelle verlinken.',
+            'app_url' => 'Anwendungs URL',
+            'timezone_help' => 'Die Zeitzone sollte einer von PHP\'s unterstützten Zeitzonen entsprechen. Solltest du dir unsicher sein, bitte verweise auf http://php.net/manual/de/timezones.php .',
```

Comment:
"to reference something" in German is "etwas entnehmen", but because that sounds awful in German, I suggest:

"...unsicher sein, informiere dich bei..."